app/helpers/common_parameters_helper.rb

```diff
@@ -19,8 +19,12 @@ def parameter_value_content(id, value, options)
     content_tag(:span, options[:popover], :class => "input-group-addon") + lookup_key_field(id, value, options)
   end
 
-  def use_puppet_default_help link_title = nil, title = _("Use Puppet default")
-    popover(link_title, _("Do not send this parameter via the ENC.<br>Puppet will use the value defined in the manifest."), :title => title)
+  def omit_help(
+      link_title = nil,
+      title = _("Omit prameter from classification"),
+      body = _("Foreman will not send this parameter in classification output.")
```

Comment:
+1 for removing the parameters because of my comment above, we keep it generic for reuse so it doesn't make sense to have it.
The only possible reason to leave it is if you do want to change the text per view.